openhantek/src/docks/VoltageDock.cpp

```diff
@@ -129,3 +154,38 @@ void VoltageDock::setUsed(ChannelID channel, bool used) {
     QSignalBlocker blocker(channelBlocks[channel].usedCheckBox);
     channelBlocks[channel].usedCheckBox->setChecked(used);
 }
+
+void VoltageDock::setProbeGain(ChannelID channel, unsigned int probeIndex) {
+    if (channel < 0 ||channel >= spec->channels) return ;
```

Comment:
ChannelID is a typedef to unsigned (and channels will always be a positive number), therefore you don't need to check for "channel < 0".